packages/sdk-react/src/components/receivables/InvoicesTable/InvoicesTable.tsx

```diff
@@ -220,7 +238,7 @@ const InvoicesTableBase = ({ onRowClick }: InvoicesTableProps) => {
               flex: 0.5,
             },
             {
-              field: 'fulfillment_date',
```

Comment:
Yo, no we can't. `fulfillment_date` is a date when the invoice has been created. But `due_date` is a date for the customer when they have to pay for this invoice.

As an example of a salary invoice. You create an invoice to Monite 29th of May and set `due_date` as `01.06` for Monite.

Reply:
But this is a matter of what to show. In my humble opinion `due_date` is much more important than `fulfillment_date` so I think we could change it.
Because we don't have a product owner right now I think we could change it with no hesitation